roles/openshift_logging_kibana/tasks/main.yaml

```diff
@@ -95,6 +95,20 @@
     - port: 443
       targetPort: "oaproxy"
 
+- name: Set memcached service
```

Comment:
I would prefer not to add yet another role, but if its a component in logging we intend to rely on and use outside of kibana then it should have its own role.